docs/docsite/rst/network/user_guide/faq.rst

```diff
@@ -0,0 +1,34 @@
+
+* Use ``ProxyCommand`` only if you absolutely must
+
+Network modules support the use of a jump host with the ``ProxyCommand`` parameter. However, when you use a jump host, Ansible must open a new SSH connection for every task, even if you are using a persistent connection type (``network_cli`` or ``netconf``). To maximize the performance benefits of the persistent connection types introduced in version 2.5, avoid using jump hosts whenever possible.
+
+* Set ``--forks`` to match your needs
+
+Every time Ansible runs a task, it forks its own process. The ``--forks`` parameter defines the number of concurrent tasks - if you retain the default setting, which is ``--forks=5``, and you are running a playbook on 10 hosts, five of those hosts will have to wait until a fork is available. Of course, the more forks you allow, the more memory and processing power Ansible will use. Since most network tasks are run on the control host, this means your laptop can quickly become cpu- or memory-bound. 
+
+Why is my output sometimes replaced with "********"?
+====================================================
+
+Ansible replaces any string marked ``no_log``, including passwords, with ``********`` in Ansible output. This is done by design, to protect your sensitive data. Most users are happy to have their passwords redacted. However, Ansible replaces every string that matches your password with ``********``. If you use a common word for your password, this can be a problem. For example, if you choose ``Admin`` as your password, Ansible will replace every instance of the word ``Admin`` with ``********`` in your output. This may make your output harder to read. To avoid this problem, select a password that will not occur elsewhere in your Ansible output. 
+
+Why do the ``*_config`` modules always return ``changed=true`` with short-form commands?
+========================================================================================
+
+When you are issuing commands directly on a network device, you can use long-form and short-form commands interchangeably. For example ``interface`` and ``int`` have the same effect. Ansible's ``*_command`` modules return the same results no matter which form of the command you use, because they run those commands through the network OS. However, behind the scenes, the network OS will convert short-form commands into long-form before committing the state of the configuration. Ansible's ``*_config`` modules check the difference between the command provided and the existing configuration. Because existing configuration is always expressed long-form, using the short-form commands with the ``*_config`` modules will always return ``changed=true``, even when the state of the configuration matches the short-form command given. To avoid this problem and make your playbooks more efficient, use long-form commands with the ``*_config`` modules.
```

Comment:
If we change to headings (rather than bullet points) I wonder if we could split this block of text up a bit to increase readability.